LICENCE.rst

```diff
@@ -1,8 +1,10 @@
 GrantNav
 ========
 
+Note: This license doesn't apply to the 360Giving theme and related assets (such as the logo).
```

Comment:
My intent was to cover everything that we copied from the 360Giving website, as we don't have permission to release it under a FOSS license.

The next question then is what of that is copyright-able?

I think the logo is:
https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo.png
https://github.com/OpenDataServices/grantnav/blob/master/grantnav/frontend/static/images/360logo_invert.png

Beyond that I was worried about the "look and feel" of the theme. That the choice of colours / font / layout might constitute a copyright-able artistic work. I've done some brief googling, and I'm not sure if that is true.